jobs/V3/basic_company_sample.rb

```diff
@@ -0,0 +1,211 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def job_discovery_generate_company display_name:, headquarters_address:
+	# [START generate_company]
+	# display_name         = "Your company display name"
+	# headquarters_address = "Your company headquarters address"
+
+	require "google/apis/jobs_v3"
+	require "securerandom"
+
+	jobs = Google::Apis::JobsV3
+	external_id = "externalId: #{display_name} #{SecureRandom.hex}"
```

Comment:
I still don't agree with `SecureRandom.hex`. I think this can be passed in with the `display_name`